README.md

```diff
@@ -8,7 +8,7 @@ What `swift-log` provides today can be found in the [API docs][api-docs]. At thi
 
 ---
 
-**NOTE**: You will need Xcode 10.2 beta 4 or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
+**NOTE**: You will need Xcode 10.2 beta 4, [Xcode 10.2](https://itunes.apple.com/us/app/xcode/id497799835) or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
```

Comment:
his this a good link, won't change?

Reply:
Looks like it to me:

> These URLs are great for linking to your app or company page in the App Store or Mac App Store from your website or inside your app, or anywhere you don't need to type or speak the URL. As a rule, always use these standard links when driving users to your app from your digital marketing communications.


https://developer.apple.com/library/archive/qa/qa1633/_index.html

Reply:
Ah, yah it's also linked from https://swift.org/download/#releases for the Xcode link.